mail_operating_unit/models/mail_template.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2024 Camptocamp SA
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo import fields, models
+
+
+class MailTemplate(models.Model):
+    _inherit = "mail.template"
+
+    operating_unit_id = fields.Many2one("operating.unit", string="Operating Unit")
+
+    def send_mail(
+        self, res_id, force_send=False, raise_exception=False, email_values=None
+    ):
+        """Include in the email values the alias domain ID of the current OU if any"""
```

Comment:
```suggestion
        # Include in the email values the alias domain ID of the current OU if any
```
Using `"""` will erase the previous doctring of the method with yours, I don't think this is what we want :slightly_smiling_face: